modules/client/src/tokenVoting/internal/client/methods.ts

```diff
@@ -345,6 +353,73 @@ export class TokenVotingClientMethods extends ClientCore
     };
   }
 
+  public async *wrapTokens(
+    params: WrapTokensParams,
+  ): AsyncGenerator<WrapTokensStepValue> {
+    const signer = this.web3.getConnectedSigner();
+    if (!signer) {
+      throw new NoSignerError();
+    } else if (!signer.provider) {
+      throw new NoProviderError();
+    }
+    if (!isAddress(params.wrappedTokenAddress)) {
+      throw new InvalidAddressError();
+    }
+    const wrappedErc20Contract = GovernanceWrappedERC20__factory.connect(
+      params.wrappedTokenAddress,
+      signer,
+    );
+
+    const account = await signer.getAddress();
+
+    const tx = await wrappedErc20Contract.depositFor(
+      account,
+      BigNumber.from(params.amount),
```

Comment:
Nope, the contracts acept a `BigNumberish` but unfortunatielly `bigint` is not a part of the type `BigNumberish` so either you change it to string or to `BigNumber` I chose the later because itt's how its done in the rest of the code